cla-backend-go/swagger/common/properties/project-name.yaml

```diff
@@ -0,0 +1,6 @@
+type: string
+description: The name of the project
+example: "Cloud Foundry Foundation"
+pattern: '^([\w\d\s\-\,\./]+){2,255}$'
```

Comment:
With minLength and MaxLength set , could {2,255} be repetition?

Reply:
Actually, I think we duplicated this because, as I recall, the {2,255} didn't work as expected for some reason. Needs investigation....